sale_quotation_number/models/sale_order.py

```diff
@@ -4,17 +4,22 @@
 # © 2020 Manuel Regidor  <manuel.regidor@sygel.es>
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
 
-from odoo import api, models
+from odoo import api, fields, models
 
 
 class SaleOrder(models.Model):
     _inherit = "sale.order"
 
+    quotation_seq_used = fields.Boolean(
+        string="Quotation Sequence Used", default=False, copy=False, readonly=True
+    )
+
     @api.model
     def create(self, vals):
         if self.is_using_quotation_number(vals):
             sequence = self.env["ir.sequence"].next_by_code("sale.quotation")
             vals["name"] = sequence or "/"
+            vals["quotation_seq_used"] = True
```

Comment:
mejor realizarlo en un write.

Reply:
Why should we do create+write when we can do just create and set the value to True when necessary?